drape_frontend/frontend_renderer.cpp

```diff
@@ -762,6 +765,17 @@ void FrontendRenderer::AcceptMessage(ref_ptr<Message> message)
       }
       break;
     }
+  case Message::Type::SetMapLanguageCode:
+    {
+      ref_ptr<SetMapLanguageCodeMessage> const msg = message;
+
+      if (m_mapLanguageCode != msg->MapLanguageCode())
+      {
+        m_mapLanguageCode = msg->MapLanguageCode();
+        m_forceUpdateScene = true;
```

Comment:
Is this whole drape code/message really needed? I don't see that `m_mapLanguageCode` is used anywhere.

There should be a way to initiate map redrawing in the framework, @vng right?

Reply:
⬆️